extensions/analyticsdx-vscode-templates/test/vscode-integration/schemas/layoutSchema.test.ts

```diff
@@ -56,6 +56,7 @@ describe('layout-schema.json hookup', () => {
         pages: [
           {
             title: '',
+            type: 'Configuration',
```

Comment:
Needed to add this to a bunch of the existing test layout.json's since it's now required.